modules/oadp-gcp-wif-cloud-authentication.adoc

```diff
@@ -17,10 +17,9 @@ Workload identity federation handles encrypting and decrypting certificates, ext
 Google workload identity federation is available for OADP 1.3.x and later.
 ====
 
-[NOTE]
-====
-For backing up volumes, OADP on GCP with Google workload identity federation authentication supports only CSI snapshots.
-====
+For backing up volumes, OADP on GCP with Google workload identity federation authentication only supports CSI snapshots.
```

Comment:
They are defined in the document - https://docs.openshift.com/container-platform/4.15/backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.html

Reply:
Do as you think best, but any page we provide may be the first page on which a reader encounters the abbreviation/acronym.

Reply:
Yes this is a module included in a assembly. The abbreviations are included in modules included earlier in the assembly or in the assembly itself.

The first line on that page is `You install the OpenShift API for Data Protection (OADP) with Google Cloud Platform (GCP)`. Later in the same assembly is the line `If you use Container Storage Interface (CSI) snapshots`

Given that this is the final module included in the assembly, it would look odd to repeat the abbreviations and acronyms.